screens/MessageDetailsScreen.js

```diff
@@ -15,6 +15,7 @@ export default function MessageDetailsScreen(props) {
     <View style={styles.container}>
       <ScrollView
         style={styles.Container}
```

Comment:
A lot a right: 1s

Reply:
If you don't have that line, the scrollbar is in the middle of the page on the ios simulator